docs/installation.md

```diff
@@ -58,6 +58,16 @@
     argocd-diff-preview --help
     ```
 
+    ## Install binary with [aquaproj/aqua](https://github.com/aquaproj/aqua)
+    `argocd-diff-preview` also supports installation by aqua. See [documentation](https://aquaproj.github.io/docs/install) for aqua setup instructions.
+
+    ```bash
+    aqua init
+    aqua g -i dag-andersen/argocd-diff-preview
+    aqua i
+    argocd-diff-preview --help
+    ```
+
```

Comment:
This was now moved to its own section 
```suggestion
```

Reply:
Thanks! I applied your suggestion on below commit.
https://github.com/dag-andersen/argocd-diff-preview/pull/54/commits/6f19ca9edc9fc5c4fa5ad1bc89ac1803e65a9e97